lib/auth/proto/auth.proto

```diff
@@ -73,6 +74,26 @@ message WatchKind {
     string Name = 3  [(gogoproto.jsontag) = "name"];
 }
 
+// Set of certificates corresponding to a single public key.
+message CertSet {
+    // SSH X509 cert (pem encoded).
+    bytes Ssh = 1 [(gogoproto.jsontag) = "ssh,omitempty"];
+    // TLS X509 cert (pem encoded).
+    bytes Tls = 2 [(gogoproto.jsontag) = "tls,omitempty"];
+}
+
+// User certificate signing request, used by `tsh sign`.
+message UserCertsRequest {
+    // Pulic key to be signed.
```

Comment:
`Key is a public key to be signed`, all comments in go according to the guidelines should start with the name of the property/function being documented.